README.md

```diff
@@ -1,18 +1,155 @@
-# ECC Python Cookiecutter Template
+# Python Cookiecutter Template
 
 This is a Cookiecutter template for creating a Python project with Poetry,
 including configurations for various project settings.
 
 ## Usage
 
-1. **Install Cookiecutter** (if you haven't already):
+1. **Install cruft** (if you haven't already):
 
 ```sh
-pip install cookiecutter
+pip install cruft
 ```
 
 2. **Generate a New Project**:
 
 ```sh
-cookiecutter gh:elixir-cloud-aai/cookiecutter-python
+cruft create https://github.com/elixir-cloud-aai/cookiecutter-python.git
 ```
+
+## Configurations
+
+### Package manager - Poetry
+
+This project uses [Poetry](https://python-poetry.org/) as a package manager.
+Check out the commands at the
+[official documentation](https://python-poetry.org/docs/cli/).
+
+### Linters and formatters - Ruff
+
+To lint and format `Python` code files, it uses
+[Ruff](https://docs.astral.sh/ruff), the default configuration is set in the
+`pyproject.toml` file.
+
+```toml
+select = [
+  "B", # flake8-bugbear
+  "E", # pycodestyle
+  "F", # Pyflakes
+  "I", # isort
+  "PL", # pylint
+  "SIM", # flake8-simplify
+  "UP", # pyupgrade
+]
+```
+
+To configure it to your needs, refer to the
+[rules documentation](https://docs.astral.sh/ruff/rules/), and for formatter
+configuration, refer to the
+[configuration documentation](https://docs.astral.sh/ruff/formatter)
+
+### Spell checker - typos
+
+If you want to ignore certain words, add them to the `pyproject.toml` file,
+under the `tool.typos.default.extend-words` key.
+
+```toml
+[tool.typos.default.extend-words]
+mke = 'mke'
+```
+
+For further configuration, refer to the [typos docs](https://pypi.org/project/typos/).
+
+### Static type checker - mypy
+
+Change the configuration in `mypy.ini` file, for further information refer to
+the [documentation](https://mypy.readthedocs.io/en/stable/config_file.html).
```

Comment:
Use `Mypy` (capitalized), unless you are talking about the executable (then use `mypy`, formatted with "`". Also, please include link to main Mypy page.